testutil/compose/compose/alert.go

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/log"
+	"github.com/obolnetwork/charon/app/z"
+)
+
+// startAlertCollector starts a server that accepts alert webhooks until the context is closed and returns
+// a channel on which the received webhooks will be sent.
+func startAlertCollector(ctx context.Context, port int) (chan []byte, error) {
+	l, err := net.Listen("tcp", fmt.Sprintf("0.0.0.0:%d", port))
+	if err != nil {
+		return nil, errors.Wrap(err, "new listener")
+	}
+
+	bodies := make(chan []byte)
```

Comment:
nit: probably a better variable name 😅

Reply:
💀

Reply:
Once I have the json format of the alerts, I'mm probably make it into a proper types struct.

Reply:
cool